### PR TITLE
Single Player Input Actions

### PR DESCRIPTION
**Pull request by ParthCv**

### CODE REVIEW

File: Assets/Scripts/Player.cs [hunk 1]
```diff
@@ -2,22 +2,39 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.InputSystem;
+using Math = System.Math;
 
 public class Player : MonoBehaviour
 {
-    [SerializeField]
-    private InputActionReference m_MoveUpAction;
+    // [SerializeField]
```
Comment (le-rich) [inline Assets/Scripts/Player.cs]: No dead code in Master Please, remove this comment

**Review by le-rich — CHANGES_REQUESTED** [review]

**Comment by normanlim** [comment]
pls clean up code

**Comment by ParthCv** [comment]
Done Changes

**Review by le-rich — APPROVED** [review]
Thnx daddy

**Pull request merged**